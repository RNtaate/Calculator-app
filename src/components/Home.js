import React from 'react';

const Home = () => {
  const statement = 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Excepturi modi doloribus necessitatibus, sint, quod repellendus tempora cupiditate libero, voluptatum similique reprehenderit eius? Earum asperiores quo alias id animi, perferendis corrupti!';

  return (
    <div style={{
      padding: '20px',
    }}
    >
      <h2 style={{
        fontSize: '35px',
        padding: '20px 0',
      }}
      >
        Welcome to our page!
      </h2>
      <p>{statement}</p>
      <br />
      <p>{statement}</p>
    </div>
  );
};

export default Home;
