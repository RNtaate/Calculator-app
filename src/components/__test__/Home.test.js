import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Home from '../Home';

it('renders Home component without crashing', () => {
  const myDiv = document.createElement('div');
  ReactDOM.render(<Home />, myDiv);
});

it('It matches Home component snaphsot', () => {
  const tree = renderer.create(<Home />);
  expect(tree).toMatchSnapshot();
});
