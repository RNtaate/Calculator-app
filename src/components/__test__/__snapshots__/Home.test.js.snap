// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It matches Home component snaphsot 1`] = `
<div
  style={
    Object {
      "padding": "20px",
    }
  }
>
  <h2
    style={
      Object {
        "fontSize": "35px",
        "padding": "20px 0",
      }
    }
  >
    Welcome to our page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Excepturi modi doloribus necessitatibus, sint, quod repellendus tempora cupiditate libero, voluptatum similique reprehenderit eius? Earum asperiores quo alias id animi, perferendis corrupti!
  </p>
  <br />
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Excepturi modi doloribus necessitatibus, sint, quod repellendus tempora cupiditate libero, voluptatum similique reprehenderit eius? Earum asperiores quo alias id animi, perferendis corrupti!
  </p>
</div>
`;
